create_derivatives/tests.py

```diff
@@ -77,3 +77,38 @@ def test_run(self, mock_get_object, mock_init):
     def tearDown(self):
         for f in Path(settings.SRC_DIR).iterdir():
             f.unlink()
+
+
+class PDFMakerTestCase(TestCase):
+
+    def setUp(self):
+        tmp_path = Path(settings.TMP_DIR)
+        if not tmp_path.exists():
+            tmp_path.mkdir(parents=True)
+        self.bag_id = "3aai9usY3AZzCSFkB3RSQ9"
+        self.set_up_bag("unpacked_bag_with_jp2", self.bag_id)
+
+    def set_up_bag(self, fixture_directory, bag):
+        """Adds an uncompressed bag fixture to the temp directory and database"""
+        bag_path = str(Path(settings.TMP_DIR, bag))
+        if not Path(bag_path).exists():
+            shutil.copytree(Path("create_derivatives", "fixtures", fixture_directory, bag), bag_path)
+            Bag.objects.create(
+                bag_identifier="sdfjldskj",
+                bag_path=bag_path,
+                origin="digitization",
+                as_data="sdjfkldsjf",
+                dimes_identifier=bag,
+                process_status=Bag.JPG2000)
```

Comment:
I think we'll want to look comprehensively at tests in the next sprint, but I would argue that we should move towards storing dB state in a fixture, rather than creating it manually.

Reply:
The things I'm seeing in tests are:
- setting database state using fixtures rather than creating objects in tests.
- getting the right binaries in the right place (there are a couple of different representations of bags that might need to be accounted for).

Reply:
Okay--is that second piece part of changing how we set up the database in tests, or generally reviewing the directory structure of fixtures (or something else)?

Reply:
I think it's probably fixtures and a helper for moving files.